en/core-libraries/time.rst

```diff
@@ -298,86 +352,96 @@ Comparing With the Present
 .. php:method:: isThisMonth()
 .. php:method:: isThisYear()
 
-You can compare a ``Time`` instance with the present in a variety of ways::
+You can compare a ``FrozenTime`` instance with the present in a variety of ways::
 
-    $time = new Time('2014-06-15');
+    $time = new FrozenTime('+3 days');
 
-    echo $time->isYesterday();
-    echo $time->isThisWeek();
-    echo $time->isThisMonth();
-    echo $time->isThisYear();
+    var_export($time->isYesterday());
+    var_export($time->isThisWeek());
+    var_export($time->isThisMonth());
+    var_export($time->isThisYear());
```

Comment:
```suggestion
    debug($time->isYesterday());
    debug($time->isThisWeek());
    debug($time->isThisMonth());
    debug($time->isThisYear());
```

I don't think `var_export` is what we want here as we're not trying to create executable code. `debug()` will also print the true/false properly and is more inline with the rest of the cake docs.